### ISSUE TITLE
Feature Request: WPCLI support 

### ISSUE THREAD

**Issue by bacoords** [WordPress/create-block-theme#688]
For developers who want to continually save their changes for version control purposes, doing this through the UI can be cumbersome. Beyond just "save", I think many of the commands in CBT could also have complementary WPCLI commands, like making a variation or exporting the theme.

## "Save" Example

I've created a very crude, messy example of what it could look like to have a "save" command like this 

`wp cbt save` or `wp cbt save --saveTemplates` et cetera

that a developer could run at any time to push their changes to their local theme or automate somehow.

This is not opened as a PR because it's not meant to be good code! It's just a janky working demo to see if the idea felt helpful in practice. A real PR would need to have some sort of abstraction layer and better loading of the class, right now I'm just duplicating code from the rest API endpoints.

https://github.com/bacoords/create-block-theme/tree/add/cli-export-command

**Comment by colorful-tones** [comment]
I can see this being quite handy. 👍

**Comment by webdados** [comment]
I also agree this should be implemented, as stated here: https://wordpress.org/support/topic/wp-cli-commands-6/

**Comment by pbking** [comment]
Yeah, that's pretty slick.  I like this idea and honestly the stashed code is a pretty good start.

**Comment by cristi-d** [comment]
👍 This would make for a great feature that would enable users to automate their deployments.

**Comment by bph** [comment]
Thanks @bacoords for the original prototype — it's been a useful starting point for thinking through the shape of
  this. Thank you to all who posted encouraging comments. 

  I've opened #828 as a tracking issue with a proposed four-PR plan: extract a small service layer from the existing
  REST handlers (save and export) first, then ship wp cbt save and wp cbt export as thin CLI wrappers over those
  services. The goal is to avoid the REST/CLI duplication the prototype already flags, and to land PHPUnit coverage on
  the existing endpoints as a side benefit.

  Variation, clone, child-theme, blank-theme, and reset are deliberately deferred to follow-ups once the pattern is
  established.

  Discussion on scope, sequencing, or naming is very welcome on #828 before any PRs are opened.

**Comment by bacoords** [comment]
Thanks @bph for this! - The abstraction into the separate service makes the most sense. Once that's done, we could even expose the commands as Abilities for an MCP.
